frontend/src/console/components/NoticeBoard/NoticeBoard.tsx

```diff
@@ -198,12 +198,17 @@ const NoticeBoardComponent: React.FC<Props> = (props: Props) => {
     <div className="explanation-container">
       <p>
         <WrappedMessage id="explanation" messages={messages} />
-        <br />
-        <br />
-        <WrappedMessage id="blogpost_text" messages={messages} />
-        <a href="https://opencraft.com/blog">
-          <WrappedMessage id="blogpost_link" messages={messages} />
-        </a>
+        <p />
```

Comment:
Could we just separate these into two paragraphs? It looks odd to have an empty paragraph inside a paragraph.

Reply:
That was the linter. Sure!